### DISCUSSION TITLE
Listeners intermittent connection -mitigation exists?

### DISCUSSION THREAD

**Discussion by natoleal** [AzuraCast/AzuraCast#4662] [General]
Hello all, 

I would like to know if azuracast provides any solution for listeners' intermittent connection?
 My main issue is that some of my listeners have an intermittent connection, so once their internet lags the music simply stops and it is unable to re-establish the connection. I assume some kind of buffer, even it is it some seconds would already "help" mitigating.

Anyways, I would like to know how do you all handle this particular issue?

thanks in advance.

**Comment by NixAshton** [comment]
Manage Station > Edit Profile > AutoDJ > Scroll down to "DJ/Streamer Buffer Time (Seconds)" add some more seconds to the buffer.. worked for us.

**Reply by natoleal** [reply]
how does it exactly work? meaning it will be always xx seconds behind the stream, or does it mean it will buffer xx seconds and in case of intermittent internet it has this xx seconds "backup"?  and the second question, which value do you recommend having?

**Reply by NixAshton** [reply]
We have 15 seconds set. From our testing when you connect there is a +15 to +20 second delay from true broadcast, dodgy connections in the background will then have up to 15 seconds to boost up the buffer or realign the connection resulting in the end user not noticing any issues. I'm not a true technical scientist and could be completely wrong but this was an issue we had during trials and we found the issue was resolved by adding the 15 seconds into the stream buffer rate using the method originally mentioned above. Hope it helps!

**Reply by NixAshton** [reply]
I'll add that we had issues from primarily Android end users until we beefed up the 15 second buffer rate.. Live input streaming.
